anchore_engine/services/catalog/api/controllers/archives.py

```diff
@@ -86,19 +85,23 @@ def list_archives():
     """
     try:
         with session_scope() as session:
-            imgs = db_archived_images.summarize(session)
-            rules = session.query(ArchiveTransitionRule).filter_by(account=ApiRequestContextProxy.namespace()).all()
+            imgs = db_archived_images.summarize(session) or []
+            rules = session.query(ArchiveTransitionRule).filter_by(
+                account=ApiRequestContextProxy.namespace()).all() or []
             rule_count = len(rules)
-            newest = max(map(lambda x: x.last_updated, rules))
+            newest = ''
```

Comment:
Does the current time make sense here if the rules_count == 0? I'd like to keep the response have a consistent type (string, date formatted in this case). That way the field can be specified to have a specific format regex that will always hold.

Reply:
nvm. I don't think the current time makes sense here.